### DISCUSSION TITLE
Publishing docker image for multiple platforms is disabled in Insiders, why?

### DISCUSSION THREAD

**Discussion by purpleclay** [squidfunk/mkdocs-material#7847] [Q&A] [answered]
Is there a reason why the Qemu and multi-platform steps are commented out in the Insiders `build.yml` file? I have an ARM Mac and cannot pull from my GHCR repository. Is the intention that we should uncomment those lines? I noticed in the non-insiders version that a multi-platform image is published. Thanks.

**Comment by squidfunk** [comment, marked as answer]
Yes, it's disabled because it's not relevant for everybody, and the [QEMU build takes 10 times as long](https://github.com/squidfunk/mkdocs-material/actions/runs/12223122238/job/34094252176) ☺️ it's just more economical, and we want to provide a good and cheap default, since build cost on GitHub private repositories are very relevant.

You can just comment it in and commit it on your fork.

**Reply by purpleclay** [reply]
Thanks for the super fast response. Ah yes, I have never had a private project on GitHub before, so didn't register the fact there would be costs 😄
